source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,356 @@
+and re-enabling Device Sync. There are some additional effects of migration:
+
+- In the App Services UI, the Rules and Schema pages will be read-only.
+- Your App will experience a period of read-only Sync and then a 
+  period of downtime.
+- The new Flexible Sync configuration will set 
+  :ref:`client max offline time <client-maximum-offline-time>` to 30 days.
+- Storage in your Atlas cluster used by Device Sync metadata is temporarily doubled.
+  Partition-Based Sync metadata and Flexible Sync metadata must temporarily co-exist. 
+  Depending on the size of your synced data, this could affect billing every time you
+  migrate.
+
+**To migrate from Partition-Based Sync to Flexible Sync:** 
+
+#. In the App Services UI, go to your App and select the Device Sync page.
+#. At the top of the page, select :guilabel:`[Migrate to Flexible Sync]`.
+   
+   .. image:: /images/begin-migration.png
+      :alt: Begin a migration
+   
+#. Read the information about migrating, then select 
+   :guilabel:`Next: Migration Requirements`.
+#. Ensure you meet the migration requirements, the select 
+   :guilabel:`Start Migration`.
+
+.. note:: Temporary Sync Disruption
+
+   Clients will not be able to sync their local writes until the migration has
+   completed.
+
+After a migration is completed, Partition-Based Sync client apps can communicate
+with your App's Flexible Sync backend. You can disable and re-enable Flexible Sync
+and client apps will still communicate with your backend.
+
+If you re-enable Partitition-Based Sync you will lose the "migrated" status.
+This means you'll need to to another migration if you want to use Flexible Sync
+on the backend with Partition-Based Sync client apps.
+
+Migrate Permissions and Rules
+`````````````````````````````
+
+Your App's permissions and rules will be automatically migrated if the Partition-Based
+Sync rules have direct App Services Rules equivalents. This will override
+previously-defined App Services Rules.
+
+In step 4 above, you will see if your permissions and rules can be automatically
+migrated. On the :guilabel:`Review Migration Process` step in the App Services UI,
+look for :guilabel:`Rules & Permisssions`.
+
+Some Partition-Based Sync rule strategies can't translate directly to App Services
+Rules. You may need to manually migrate permissions that include:
+
+- :json-operator:`%function` operator.
+
+  :ref:`Function rules <pbs-function-rules>` are not compatible with Flexible Sync 
+  and cannot be migrated.
+
+- :json-expansion:`%%partition` expansion.
+
+  App Services Rules do not have an equivalent expansion for  
+  :json-expansion:`%%partition`, so it cannot be migrated.
+
+- ``%or``, ``%not``, ``%nor``, ``%and`` expansions.
+
+  These permissions may work, but there is enough nuance that you should test them
+  to ensure expected behavior. Testing new permissions won't work on the App
+  you are migrating. You'll need a new app to test manually-migrated permissions.
```

Comment:
There's a little ambiguity here. Are we advising people to test manually-migrated permissions in a new FS app, and _then_ apply them manually here so they can complete the migration?

Reply:
Yeah, this section was tricky to write. It's a "probably will work, but we can't guarantee it" situation. And, yes, your read is correct. I'll edit to make that clearer.

Reply:
The divergence is mainly with the permission behavior in non-sync services. An `%or` expression in an app services rule that works in flexible sync will not work outside of sync and vice versa 🥴. This is a known issue that's currently being addressed, so once that's fixed we _may_ be able to migrate those scenarios automatically (can't promise that though!)